__tests__/app.test.js

```diff
@@ -113,4 +113,68 @@ describe('GET: /api/reviews/:review_id', () => {
                 })
         })
     })
+})
+
+describe('GET: /api/reviews', () => {
+    it('responds with correct status code 200', () => {
+        return request(app)
+            .get('/api/reviews')
+            .expect(200)
+    })
+    it('gives an array of objects with the following properties', () => {
+        return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then((res) => {
+                const body = res.body.reviews
+                expect(body).toBeInstanceOf(Array)
```

Comment:
Good testing here. You could be more thorough in checking the response data types by checking the length of the returned array of reviews and using a **forEach** to assert that all the properties are of the correct data type. Let me know if you need more info on this 👍